packages/data-context/test/unit/sources/ProjectDataSource.spec.ts

```diff
@@ -226,7 +226,7 @@ describe('getDefaultSpecFileName', () => {
     it('replace * for filename', () => {
       const defaultFileName = getDefaultSpecFileName('cypress/e2e/*.spec.ts', 'e2e')
 
-      expect(defaultFileName).to.eq('cypress/e2e/filename.spec.ts')
+      expect(defaultFileName).to.eq('cypress/e2e/spec.spec.ts')
```

Comment:
`spec.spec.ts` looks a little funky to me.... thoughts on something like `integration-test.spec.ts`?